src/app/modules/courses/components/course-add-edit/course-add-edit.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="card">
-  <form [formGroup]="form" (submit)="save()">
+  <form [formGroup]="form">
```

Comment:
you should leave submit event here. Otherwise you will loose default browser forms submitting functionality (for example submit on enter click)